poetry.toml

```diff
@@ -0,0 +1,2 @@
+[virtualenvs]
```

Comment:
What's the difference in usage for the `poetry.toml` and the `pyproject.toml`? They can't be consolidated now that `pypa` was removed from the GitHub Action?

Reply:
I wondered about that. The poetry docs specifically mention that the local configuration is stored in `poetry.toml` and not in `pyproject.toml`. I agree that it would be nice to have all the config stuff in a single place.

Reply:
I think the reason for this is that the `poetry.toml` file could potentially include sensitive information that should not be checked into a repo (I think it's possible to store credentials). At the moment, I'm only specifying a local virtual environment. I like to do this because VS Code, for example, will automatically detect it and prompt you to use it for your project. And if you've installed all the dependencies, you'll be able to use the same local version of `black`, `isort`, etc. in your editor.